code/modules/reagents/Chemistry-Holder.dm

```diff
@@ -955,16 +955,16 @@ trans_to_atmos(var/datum/gas_mixture/target, var/amount=1, var/multiplier=1, var
 
 
 //written for ethylredoxrazine, but might be fun for turning water into wine or something
-/datum/reagents/proc/convert_some_of_type(var/datum/reagent/convert_from_type, var/datum/reagent/convert_to_type,var/convertall = FALSE)
+/datum/reagents/proc/convert_some_of_type(var/datum/reagent/convert_from_type, var/datum/reagent/convert_to_type,var/convert_amount = 0)
 	var/total_amount_converted = 0
 
 	for(var/datum/reagent/itsareagent in reagent_list)
 		if(istype(itsareagent, convert_from_type))
 			var/amount_to_convert
-			if(!convertall)
+			if(convert_amount == 0)
```

Comment:
This doesn't make sense. If you want to have a default value, which I don't recommend, you would write it in the arguments like `var/convert_amount = 2 * REM`.
But what I recommend instead is to not provide a default value at all, require an amount to be specified. Adjust callers of this proc accordingly

Reply:
ok that makes sense